experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -42,9 +42,6 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 	private readonly editManager: EditManager<TEditor, TChange, ChangeFamily<TEditor, TChange>>;
 	private readonly summarizables: readonly Summarizable[];
 
-	/** Iff false, calls to `submitOp` will have no effect */
-	private submitOps = true;
```

Comment:
@noencke it looks like you added this, so curious what you think about this change. the idea is to have applyStashedOp actually submit the op if it wants it, just like resubmit does, rather than what it does now which is take metadata, as assume the op should be submitted again. i think this would be the correct change, but not positive

Reply:
Nice! Yea, SharedTree submits ops in response to change events firing from our commit graph - so we have to this little kludge to block it in this one case. Happy to see that it might be able to go. Having "op application" always be coupled with a corresponding "op submission" would match our model better, and seems a more consistent DDS API in a vacuum regardless.